www/gatsby-node.js

```diff
@@ -213,24 +213,24 @@ exports.createPages = ({ graphql, actions, reporter }) => {
 
   return new Promise((resolve, reject) => {
     const docsTemplate = path.resolve(`src/templates/template-docs-markdown.js`)
-    const blogPostTemplate = path.resolve(`src/templates/template-blog-post.js`)
-    const blogListTemplate = path.resolve(`src/templates/template-blog-list.js`)
-    const tagTemplate = path.resolve(`src/templates/tags.js`)
-    const contributorPageTemplate = path.resolve(
-      `src/templates/template-contributor-page.js`
-    )
+    // const blogPostTemplate = path.resolve(`src/templates/template-blog-post.js`)
```

Comment:
Leftover debugging/wip comments?

Reply:
This is part of temporary commit that limits amount of pages, so I can build preview for this PR on netlify ( https://github.com/gatsbyjs/gatsby/pull/12087/commits/58c6855ab458ee05a3557cec0ac871591f9dbc35 ). This commit will be removed before merge